files/fluentd/fluentd_prometheus_alerts.yaml

```diff
@@ -14,7 +14,7 @@
   - "alert": "FluentdQueueLengthBurst"
```

Comment:
@vparfonov  please remove this entire alert.  @alanconway and I had a conversation regarding the value and decided there needs to be more thought around
* What we are trying to convey and monitor
* What is actionable for someone

As-is, this alert originated in issues we had with writing to our internal store which do not seem to apply

Reply:
done